backend/automatic_checks/automatic_check_readme.md

```diff
@@ -1,46 +1,41 @@
-Disclaimer, to update the database one would need around 1TB of storage available and about 30h of computation time. 
+Disclaimer: to update the database one would need around 1TB of storage available and about 30h of computation time. 
 
```

Comment:
What is this readme file for? Can we make sure:

1. It is called README.md, so GitHub renders it.
2. We link to it from the README.md in the root, or otherwise make sure that users can find this file.

Reply:
It is supposed to explain how to get the raw datasets into the database. It is not very clear to me either why this file is not in the `docs` folder as it would make more sense, but I'll change it now. As for (2), it is linked from the root README.md